script/helpers/common.ts

```diff
@@ -385,4 +389,49 @@ export const updateLimitsAndPoolId = async (
       chain
     );
   }
+  if (
+    addresses.isAppChain
+  ) {
+    for (let sibling of siblingSlugs) {
+      const siblingConnectorAddresses: ConnectorAddresses | undefined =
+        connectors[sibling];
+      if (!siblingConnectorAddresses) continue;
+
+      const integrationTypes: IntegrationTypes[] = Object.keys(
+        siblingConnectorAddresses
+      ) as unknown as IntegrationTypes[];
+      for (let it of integrationTypes) {
+        const itConnectorAddress: string | undefined =
+          siblingConnectorAddresses[it];
+        if (!itConnectorAddress) continue;
+          connectorAddresses.push(itConnectorAddress);
+        const connectorpolId = await hookContract.connectorPoolIds(itConnectorAddress);
+        connectorPoolIds.push(connectorpolId);
```

Comment:
cant see where this array is initialized

Reply:
It is initialized at the start of the function.